config/postgres/sample_templated_config.xml

```diff
@@ -19,6 +19,22 @@
 
     <query_templates_file>data/templated/example.xml</query_templates_file>
 
+    <anon>
```

Comment:
Do you expect anything else to be part of anon? Otherwise, we might be able to drop the anonTable tag?

Reply:
It is used to handle multiple tables at once. I have renamed it for now.